### DISCUSSION TITLE
Official release of openvpn3-linux

### DISCUSSION THREAD

**Discussion by hatmaalla** [OpenVPN/openvpn3-linux#172] [General]
Hello,
I am very interested in using the dbus interface exposed by the **openvpn3-linux** client.
However, as I can see, only beta versions are available until now.
Do you have a roadmap to release the official version with the dbus interface?

**Comment by dsommers** [comment]
All releases are official.  The last few beta releases has been through stabilisation and ensuring important issues are mostly resolved.  Consider the current D-Bus interface to be stable.

**Comment by dsommers** [comment]
v20 stable is just released.  https://www.mail-archive.com/openvpn-devel@lists.sourceforge.net/msg26443.html
